[
  "test_register_userV2.py::CreateUserTestCase::test_cannot_create_duplicate_user"
]